lib/fast_inserter/fast_inserter_base.rb

```diff
@@ -94,20 +94,24 @@ def fast_insert_group(group)
 
     # Queries for the existing values for a given group of values
     def existing_values(group_of_values)
-      values_to_check = ActiveRecord::Base.send(:sanitize_sql_array, ["?", group_of_values])
-      sql = "SELECT #{@variable_column} FROM #{@table_name} WHERE #{existing_values_static_columns} AND #{@variable_column} IN (#{values_to_check})"
+      sql = "SELECT #{@variable_column} FROM #{@table_name} WHERE #{existing_values_static_columns}"
 
       # NOTE: There are more elegant ways to get this field out of the resultset, but each database adaptor returns a different type
       # of result from 'execute(sql)'. Potential classes for 'result' is Array (sqlite), Mysql2::Result (mysql2), PG::Result (pg). Each
       # result can be enumerated into a list of arrays (mysql) or list of hashes (sqlite, pg)
       results = ActiveRecord::Base.connection.execute(sql)
```

Comment:
@sgringwe we could cache this result rather than doing it each insert. possibility of other things inserting records and the cache being stale. not sure its worth the trade off